src/cmakeFindModules/OpenMVGConfig.cmake.in

```diff
@@ -83,7 +83,7 @@ set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
 # Import exported OpenMVG targets
-include(${CURRENT_CONFIG_INSTALL_DIR}/OpenMVGTargets.cmake)
```

Comment:
this file is not there anymore. it is in /CMakeFiles/Export/share/openMVG/cmake/